### PR TITLE
zh-translation: /news/releases/1.0.x/announcing-1.0.8/index.md #1535

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1535](https://github.com/servicemesher/istio-official-translation/issues/1535)

### CODE REVIEW

File: content/zh/news/releases/1.0.x/announcing-1.0.8/index.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ aliases:
...
-We're pleased to announce the availability of Istio 1.0.8. Please see below for what's changed.
+我们很高兴的宣布 Istio 1.0.8 现已正式发布。下面是更新详情。
 
 {{< relnote >}}
 
-## Bug fixes
+## Bug 修复{#bug-fixes}
 
-- Fix issue where Citadel could generate a new root CA if it cannot contact the Kubernetes API server, causing mutual TLS verification to incorrectly fail ([Issue 14512](https://github.com/istio/istio/issues/14512)).
+- 修复了 Citadel 如果无法联系 Kubernetes API 服务器可能会生成新的根 CA 的问题，导致双向 TLS 验证失败（[Issue 14512](https://github.com/istio/istio/issues/14512)）。
```
Comment (tanjunchen) [inline content/zh/news/releases/1.0.x/announcing-1.0.8/index.md]: 该问题会导致双向 TLS 验证失败（详情可参见 [Issue 14512](https://github.com/istio/istio/issues/14512)）

Comment (tanjunchen) [reply content/zh/news/releases/1.0.x/announcing-1.0.8/index.md]: 括号对吗

Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.8/index.md]: > 该问题会导致双向 TLS 验证失败（详情可参见 [Issue 14512](https://github.com/istio/istio/issues/14512)）

感谢细心 review，已修改

Comment (tanjunchen) [reply content/zh/news/releases/1.0.x/announcing-1.0.8/index.md]: OK Thanks your jobs!

**Comment by tanjunchen** [comment]
@rootsongjc 
LGTM

**Pull request merged**